akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -315,25 +330,26 @@ object FlowGraph {
     def to[I, O](j: UniformFanOutShape[I, O]): ReverseOps[I] = new ReverseOps(j.in)
 
     class ForwardOps[T](out: Outlet[T]) {
```

Comment:
can be final, but must be public. Just look at what it does, how can it be private?